src/pudl/metadata/resources/eia860.py

```diff
@@ -906,6 +906,73 @@
         "sources": ["eia860"],
         "etl_group": "eia860",
     },
+    "core_eia860__scd_generators_multifuel": {
+        "description": (
+            "Annually reported information about energy storage from EIA-860 Schedule 3. "
+            "Contains data on fuel-switching and the use of multiple fuels by surveyed generators."
+        ),
+        "schema": {
+            "fields": [
+                "utility_id_eia",
+                "utility_name_eia",
+                "plant_id_eia",
+                "plant_name_eia",
+                "state",
+                "county",
+                "generator_id",
+                "operational_status_code",
+                "technology_description",
+                "prime_mover_code",
+                "sector_name_eia",
+                "sector_id_eia",
+                "capacity_mw",
+                "summer_capacity_mw",
+                "winter_capacity_mw",
+                "current_planned_generator_operating_month",
+                "current_planned_generator_operating_year",
+                "energy_source_code_1",
+                "energy_source_code_2",
+                "multiple_fuels",
+                "cofire_fuels",
+                "cofire_energy_source_1",
+                "cofire_energy_source_2",
+                "cofire_energy_source_3",
+                "cofire_energy_source_4",
+                "cofire_energy_source_5",
+                "cofire_energy_source_6",
+                "switch_oil_gas",
+                "switch_when_operating",
+                "net_summer_capacity_with_natural_gas_mw",
+                "net_summer_capacity_with_oil_mw",
+                "net_winter_capacity_with_natural_gas_mw",
+                "net_winter_capacity_with_oil_mw",
+                "factors_that_limit_switching",
+                "storage_limits",
+                "air_permit_limits",
+                "other_limits",
+                "cofire_oil_and_gas",
+                "cofire_100_oil",
+                "max_oil_heat_input",
+                "max_oil_output_mw",
+                "fuel_switch",
+                "time_to_switch",
+                "regulatory_limits",
+                "fuel_switch_energy_source_1",
+                "fuel_switch_energy_source_2",
+                "fuel_switch_energy_source_3",
+                "fuel_switch_energy_source_4",
+                "fuel_switch_energy_source_5",
+                "fuel_switch_energy_source_6",
+            ],
+            "primary_key": [
+                "utility_id_eia",
+                "generator_id",
```

Comment:
Did you double check that this is the primary key? My guess is that `plant_id_eia` might also be part of that / we usually add it in in case there are future instances where a utility owns multiple plants with a `generator_id` 1 for example.

Reply:
Done!